library/src/scripts/headers/mebox/pieces/MeBoxDropDownItem.tsx

```diff
@@ -55,19 +55,24 @@ export default class MeBoxDropDownItem extends React.Component<IProps> {
         const classesMeBoxMessage = meBoxMessageClasses();
         const classesMetas = metasClasses();
 
-        let authors: JSX.Element[];
+        let authorList: JSX.Element[];
 
         if (this.props.type === MeBoxItemType.MESSAGE) {
             // Message
+            const { authors } = this.props;
             const authorCount = this.props.authors.length;
-            if ("authors" in this.props && authorCount > 0) {
-                authors = this.props.authors!.map((user, index) => {
-                    return (
-                        <React.Fragment key={`meBoxAuthor-${index}`}>
-                            <strong>{user.name}</strong>
-                            {`${index < authorCount - 1 ? `, ` : ""}`}
-                        </React.Fragment>
-                    );
+            if (authors && authorCount > 0) {
+                authorList = this.props.authors!.map((user, index) => {
```

Comment:
I think due to the check we can write `authors.map((user, index) => ...` here

Reply:
```suggestion
                authorList = authors.map((user, index) => {
```

Typescript notices that `authors` has been checked already, but not `this.props.authors`.